crates/store/src/state.rs

```diff
@@ -226,7 +226,7 @@ impl State {
                 .map(|(note_index, note)| {
                     let details = match note {
                         OutputNote::Full(note) => Some(note.to_bytes()),
-                        OutputNote::Header(_) => None,
+                        OutputNote::Header(_) | OutputNote::Partial(_) => None,
```

Comment:
Getting a partial note here should be an error. But this is probably not for this PR. Let's create an issue to come back to this in the future.

Reply:
Created a PR but then closed it as it is being fixed in #377